### PR TITLE
genesis: add options for warmup epochs and bootstrap stake

### PR DESCRIPTION
**Pull request by anwayde**

### CODE REVIEW

File: src/app/fdctl/config/default.toml [hunk 1]
```diff
@@ -1011,6 +1011,16 @@ dynamic_port_range = "8900-9000"
         # The amount of SOL to pre-fund each account with.
         fund_initial_amount_lamports = 50000000000000
 
+        # The amount of stake for the bootstrap validator in lamports.
+        bootstrap_stake_lamports = 5000000000
```
Comment (mmcgee-jump) [inline src/app/fdctl/config/default.toml:1015]: Is this what solana calls it? It seems pretty imprecise, what is "bootstrap"? Should this just be `vote_account_stake_lamports` ? I would expand the comment to specify this clearly as well.

**Review by mmcgee-jump — CHANGES_REQUESTED** [review]
.

File: src/app/fdctl/config/default.toml (hunk 1 above)
Comment (anwayde) [reply src/app/fdctl/config/default.toml:1015]: Yep, this is what solana calls it: https://github.com/anza-xyz/agave/blob/master/genesis/src/main.rs#L223-L229. Bootstrap validator is the term they use to refer to the validator that starts up from the genesis. It makes sense to me, but I'm okay with calling it whatever makes more sense. It sets the amount that is staked on the (vote account of the) bootstrap validator.

Comment (mmcgee-jump) [reply src/app/fdctl/config/default.toml:1015]: We can probably do better than the Solana CLI in this case. `fund_vote_account_lamports` is the clearest to me.

**Review by mmcgee-jump — APPROVED** [review]

**Pull request merged**